src/xrpld/consensus/DisputedTx.h

```diff
@@ -238,13 +302,16 @@ DisputedTx<Tx_t, NodeID_t>::updateVote(
 
     if (newPosition == ourVote_)
     {
+        ++currentVoteCounter_;
         JLOG(j_.info()) << "No change (" << (ourVote_ ? "YES" : "NO")
                         << ") : weight " << weight << ", percent "
-                        << percentTime;
+                        << percentTime
+                        << ", round(s) with this vote: " << currentVoteCounter_;
```

Comment:
I know this is also old behaviour, but would it be useful to show here `tx_.id()`, like we do below when our vote is changed ?

Reply:
> I know this is also old behaviour, but would it be useful to show here `tx_.id()`, like we do below when our vote is changed ?

Fixed.